resource/translations/ao_es.ts

```diff
@@ -37,7 +38,7 @@ Que tengas un buen día.</translation>
     <message>
         <location filename="../../src/packet_distribution.cpp" line="293"/>
         <source>Attorney Online 2</source>
-        <translation>Attorney Online 2</translation>
+        <translation></translation>
```

Comment:
what

Reply:
mission accomplished

Reply:
The client will just use the original string. So no need to copy paste.